ncats_webd/cybex_queries.py

```diff
@@ -340,8 +352,8 @@ def csv_get_open_tickets(db, ticket_severity):
                 columns=[
                     "_id",
                     "owner",
-                    "ip",
                     "hostname",
+                    "ip",
```

Comment:
I'm not specifically opposed to this change but it's unclear why you're changing the sort order piecemeal.